style.css

```diff
@@ -63,4 +63,367 @@
   .icons {
     padding-right: 1.5rem;
   }
+
+  .projects {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+    gap: 0.75;
+    width: 20.438rem;
+    height: 34.5rem;
+    background: #fff;
+    border: 0.063rem solid #dfe1e6;
+    border-radius: 1rem;
+  }
+
+  .snapshots {
+    width: 18.438rem;
+    height: 13.75rem;
+    background-color: rgb(211, 143, 95);
+    border-radius: 0.5rem;
+  }
+
+  #snap2 {
+    background-color: #d0c7d1;
+  }
+
+  #snap3 {
+    background-color: rgb(231, 119, 44);
+  }
+
+  #snap4 {
+    background-color: rgb(255, 255, 255);
+  }
+
+  .project_name {
+    margin-top: 0.2rem;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 1200;
+    font-size: 1.8rem;
+    line-height: 2.75rem;
+    margin-bottom: -0.5rem;
+  }
+
+  .details {
+    display: inline-flex;
+    flex-direction: row;
+  }
+
+  .detail {
+    display: inline-flex;
+    color: #344563;
+    padding-right: 2.6rem;
+    margin-left: -2.3rem;
+  }
+
+  .projo {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .info_block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.75rem;
+    width: 18.438rem;
+    height: 18rem;
+  }
+
+  .info {
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.938rem;
+    line-height: 1.5rem;
+    color: #344563;
+    margin-top: -1rem;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.5rem;
+    width: 13.375;
+    height: 1.5rem;
+  }
+
+  .language {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 57px;
+    background: #ebebff;
+    border-radius: 8px;
+    height: 16px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070fd;
+  }
+
+  #html {
+    width: 2.063rem;
+  }
+
+  #css {
+    width: 1.56rem;
+  }
+
+  #javascript {
+    width: 4.25rem;
+  }
```

Comment:
- [ ]  I would kindly suggest you to Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels its best practices and will help your website responsive.